README.md

```diff
@@ -1,35 +1,806 @@
-Active Choices Plug-in
-=================
+Older versions of this plugin may not be safe to use. Please review the
```

Comment:
For the image gallery plugin I removed this text, and left just the project description at the top. When it was displayed in the Wiki, it appeared as a warning/note.

But now on GitHub after the conversion, I got the impression that this warning from developers, as I was always reading it as just "not safe to use". Maybe we could move this warning to somewhere else in the file, or remove it?